pkg/webservice/handlers.go

```diff
@@ -774,8 +774,7 @@ func getApplicationsDAO(lists map[string]*scheduler.PartitionContext) []*dao.App
 	result := make([]*dao.ApplicationDAOInfo, 0, 32)
```

Comment:
Can we simplify this one too? The capacity is arbitrarily set.
The make in line 800 (new code) should also be cleaned up to get it inline with the rest of the make calls: should be similar to the make in line 790